### DISCUSSION TITLE
GR-Procedure-DirectAward no FR translation

### DISCUSSION THREAD

**Discussion by rlagrange98** [OP-TED/eForms-SDK#1075] [Q&A] [answered]
Hello,

For eforms F29 and F31 in sdk 1.9 I see that there is no FR translation for this title (Screenshot below). But the translation is present for a F33 for instance. (Screenshot 2)

![image](https://github.com/user-attachments/assets/0625aff9-69b0-48de-ae94-c4848b491034)

![image](https://github.com/user-attachments/assets/276ac9e9-711b-4346-9ec6-28ea9d98b712)

Best regards,
Romain lagrange

**Comment by encampaen** [comment, marked as answer]
Dear @rlagrange98,
the translation is available in subsequent versions of the SDK, so it could be an opportunity to update to SDK 1.13.
In the mean time we will back port the translation to SDK 1.9.
Best regards
Enrico Campanella
